data/GA/incentives.json

```diff
@@ -0,0 +1,170 @@
+[
+  {
+    "id": "GA-1",
+    "authority_type": "utility",
+    "authority": "ga-georgia-power",
+    "payment_methods": [
+      "rebate"
+    ],
+    "item": "weatherization",
+    "program": "ga_georgiaPower_homeEnergyImprovementProgram",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 250,
+      "maximum": 250
+    },
+    "owner_status": [
+      "homeowner"
+    ],
+    "short_description": {
+      "en": "Rebate for 50% of cost up to $250 for attic insulation. Addition of R-38 or greater required."
+    },
+    "start_date": 2023,
+    "end_date": 2025,
+    "bonus_available": true
```

Comment:
The data collector put some information in the "Bonus Details" column, but it all looks like it's related to how the incentives should be combined, which should go under Stacking Details instead.

1. Move the data into Stacking Details. Once Bonus Details is blank, `bonus_available` boolean will stop appearing when you rerun the script.
2. It looks like there is a maximum across several incentives, and we do have a way to represent that, but you'll need to add another file. See data/CO/incentive_relationships.json for an example of how to configure a maximum.